src/endpoints/cart.js

```diff
@@ -79,10 +79,12 @@ class CartEndpoint extends BaseExtend {
     )
   }
 
-  BulkAdd(body) {
+  BulkAdd(body, optionsField) {
+    const options = optionsField !== undefined ? optionsField: { add_all_or_nothing: false }
```

Comment:
```suggestion
  BulkAdd(body, options) {

```

Reply:
Hi @samblacklock, I thought about the suggestion you mentioned but considering it as a breaking change I went with the below code which still gives the flexibility to choose `options` while not breaking the feature for current client `  BulkAdd(body, optionsField) {
    const options = optionsField !== undefined ? optionsField: { add_all_or_nothing: false }` 
Please let me know if we want to go forward with the `breaking change` approach, thank you

Reply:
Sure, thank you for clarifying. I'll update as you suggested.